### ISSUE TITLE
[Bug] 'Switch input sources individually for each window' option doesn't work

### ISSUE THREAD

**Issue by pddshk** [BuddiesOfBudgie/budgie-desktop#598]
### Description

The option located in Settings -> Keyboard -> Input Source Switching

### Budgie version

10.9.2

### Operating System

Solus 4

### Steps to reproduce the issue

1. Go to Settings -> Keyboard -> Input Source Switching
2. Select 'Switch input sources individually for each window'
3. Have at least 2 input sources (X and Y)
4. Switch input source to X
5. Switch to any another window
6. Switch input source to Y
7. Switch window back

### Actual result

Input source is Y

### Expected result

Input source is X, as it was left before switching to another window

**Comment by fossfreedom** [comment]
yeah - have looked at this.  Will be possible to do with budgie-desktop v10.10

**Comment by dmitry-markin** [comment]
There is a workaround for this — an applet that remembers keyboard layouts per application and manually triggers layout toggling.
In Debian it's `sudo apt install budgie-keyboard-autoswitch-applet`, and you will need to add it to you budgie panel.

One caveat is that the toggling is not instant, and if you start typing too fast after switching windows the first symbol might be in wrong layout. Most of the time this is not an issue though.
